lib/time.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Pairmotron.Time do
```

Comment:
I think `Calendar` might be a better module name, since it won't conflict with Elixir's Time module like this will. And its solely dealing with weeks and dates, not times.